src/components/InjectedComponents/AutoPasteFailedDialog.tsx

```diff
@@ -85,32 +82,33 @@ export function AutoPasteFailedDialog(props: AutoPasteFailedDialogProps) {
                         </>
                     ) : null}
                     <Box marginBottom={1}></Box>
-                    <div style={{ textAlign: 'center' }}>
+                    <div>
                         {data.image ? (
                             // It must be img
-                            <Image component="img" onURL={setURL} src={data.image} width={260} height={180} />
-                        ) : null}
-                        <br />
-                        {url ? (
                             <>
+                                <Image component="img" src={data.image} width={260} height={180} />
+                                <Box marginBottom={1}></Box>
                                 <Button
-                                    variant="text"
-                                    component={Link}
-                                    download={`maskbook-encrypted-${formatDateTime(new Date()).replace(/:/g, '-')}.png`}
-                                    href={url}
-                                    startIcon={<Download />}>
-                                    {t('download')}
-                                </Button>
-                                <Button
-                                    variant="text"
-                                    component={Link}
-                                    href={url}
-                                    target="_blank"
-                                    startIcon={<OpenInBrowser />}>
-                                    {t('auto_paste_failed_dialog_image_caption')}
+                                    variant="contained"
+                                    onClick={async () => {
+                                        if (!data.image) return
+                                        await navigator.clipboard.write([
```

Comment:
Here I use `navigator.clipboard` directly since `copy-to-clipboard` does not support `navigator.clipboard.write()` yet.

Reply:
What if the browser doesn't support `navigator.clipboard.write()`? (We need to support Safari 12.2+ currently but this API is only on Safari 13) And please don't remove "download", this is the most compatible way to attach a picture across platforms.

Reply:
done